File: market/integration-test/java/ru/yandex/market/mboc/common/config/TestYtConfig.java
package ru.yandex.market.mboc.common.config;

import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Import;

@Configuration
@Import({
    ScheduledExecutorConfig.class
})
public class TestYtConfig extends YtConfig {
    public TestYtConfig(ScheduledExecutorConfig config) {
        super(config);
    }
}

File: market/test/java/ru/yandex/market/contentmapping/config/TestYtConfig.kt
package ru.yandex.market.contentmapping.config

import com.nhaarman.mockitokotlin2.mock
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import ru.yandex.inside.yt.kosher.Yt

@Configuration
class TestYtConfig : YtConfig() {
    @Bean
    override fun ytHttpApi(): Yt {
        return mock()
    }
}
